cmd/crc/cmd/delete.go

```diff
@@ -22,15 +25,31 @@ var deleteCmd = &cobra.Command{
 	},
 }
 
-func runDelete(arguments []string) {
+var clearCache bool
 
+func runDelete(arguments []string) {
 	deleteConfig := machine.DeleteConfig{
 		Name: constants.DefaultName,
 	}
+	if clearCache {
+		deleteCache()
+	}
 
 	commandResult, err := machine.Delete(deleteConfig)
 	output.Out(commandResult.Success)
 	if err != nil {
 		output.Out(err.Error())
 	}
 }
+
+func deleteCache() {
+	if globalForce {
+		os.RemoveAll(constants.MachineCacheDir)
```

Comment:
can we have a function for this?

Reply:
This could also be written as
```
if globalForce {
    yes := true
else {
    yes := input.PromptUserForYesOrNo("Do you want to delete cache")
}
if yes {
    os.RemoveAll(constants.MachineCacheDir)
}
```
Or even be factored into PromptUserForYesOrNo (with a 'force' argument), as the other user of this method has exactly the same requirements, assume yes if 'force' is true, ask if not.